Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -25,6 +26,20 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
             // profile.bookmarks.modelForRoot(self.onNewModel, failure: self.onModelFailure)
         }
     }
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: "firefoxAccountChanged:", name: NotificationFirefoxAccountChanged, object: nil)
+    }
+
+    deinit {
+        NSNotificationCenter.defaultCenter().removeObserver(self)
```

Comment:
Nit: Doesn't really matter since we're in the deinit call but since UIViewControllers observes other system notifications it's usually good practice to only remove self as an observer to the event you don't want to observe anymore instead of removing self as the observer for everything.